util/src/test/resources/org/apache/jsieve/util/EncodedSubjectEmail.eml

```diff
@@ -0,0 +1,34 @@
+Return-Path: <depetrini@libero.it>

```

Comment:
Anonymise this....

That's cool, I know all his mail internals ....

(I guess however we can keep the mail address)

Received headers, DKIM signature, X-CNFS-Analysis, x-libjamoib, User-Agent, X-CMAE-Envelope can be removed.